content/build/maven/checks.adoc

```diff
@@ -58,15 +58,13 @@ In case the check fails, you will see this kind of message:
 [INFO] ------------------------------------------------------------------------
 ----
 
-=== How to check if the ration can be increased in a module
+=== How to check if the ratio can be increased in a module
 
 According to <<id-test-coverage,the rules about the minimum test coverage ratio update>>, if a contributor increases the test coverage ratio, he/she can update the minimum ratio.
 But Maven does not give the actual value in its standard output (console).
 Instead, this value is available in the jacoco reports, in `target/site/test-coverage-ut/index.html`. 
 Here is an example:
```

Comment:
Example is missing

Reply:
example removed on purpose, will remove this sentence too